Tests/Validator/Constraints/PhoneNumberValidatorTest.php

```diff
@@ -16,39 +16,51 @@
 use libphonenumber\PhoneNumberUtil;
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber;
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumberValidator;
-use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\Validator\Context\ExecutionContext;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
 
 /**
  * Phone number validator test.
  */
-class PhoneNumberValidatorTest extends TestCase
+class PhoneNumberValidatorTest extends \PHPUnit_Framework_TestCase
```

Comment:
I would base the test on the `ConstraintValidatorTestCase` class from the Validator component: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php

Reply:
Unfortunately that class does not exists in Symfony 2.3 and the `master` branch still supports it

Reply:
Ah indeed, I forgot that. However, I'd still suggest to take a look into that class to get some inspiration on how the test could be written.

Reply:
A lot of the class you are quoting is meant to support the new API available since Symfony 2.5 (if I'm not wrong). If you have any suggestion on how I could improve this test class I would be glad to apply them, but at the moment I think it would be a lot of work for no real benefits.